feature-libs/user/profile/components/update-password/update-password-component.service.ts

```diff
@@ -78,7 +102,15 @@ export class UpdatePasswordComponentService {
     );
     this.busy$.next(false);
     this.form.reset();
-    this.routingService.go({ cxRoute: 'home' });
+
+    // sets the redirect url after login
+    this.authRedirectService?.setRedirectUrl(
```

Comment:
as the issue may indirectly imply, the logic would be similar to when someone changes the email.